nginx/nginx.conf

```diff
@@ -42,6 +43,8 @@ http {
         }
         location / {
            proxy_pass http://sentry;
+           proxy_redirect off;
+           proxy_set_header Host       $host;
```

Comment:
Move this outside of the location block.

Reply:
See https://github.com/getsentry/sentry/blob/e8b7c215d9bb8fa789584ffc827611d060f9de23/config/reverse_proxy/nginx.conf#L49. I'll try to figure out how we missed this in the first place.

Reply:
Sure, didn't know `relay` cared about this.